x-pack/platform/packages/shared/kbn-elastic-assistant-common/impl/schemas/conversations/common_attributes.gen.ts

```diff
@@ -403,15 +399,15 @@ export const ConversationCreateProps = z.object({
    * LLM API configuration.
    */
   apiConfig: ApiConfig.optional(),
-  /**
-   * Is default conversation.
-   */
-  isDefault: z.boolean().optional(),
   /**
    * excludeFromLastConversationStorage.
    */
   excludeFromLastConversationStorage: z.boolean().optional(),
   replacements: Replacements.optional(),
+  /**
+   * Set the creation date.
+   */
+  createdAt: z.string().optional(),
```

Comment:
Is there a need for clients to be able to specify `createdAt` on creation, or is this just for testing purposes?

Reply:
only for testing!

Reply:
While nice here, we may want to remove for data integrity and security purposes. We can still use the ES API's directly for testing, no?

Reply:
good point, i will work on updating my script and remove this param